src/content/blog/why-not-effect.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Why we're not using Effect"
```

Comment:
I just noticed that "Why not effect" (the title of this PR) is also a great title for the blog post